docs/resources/function_node.md

```diff
@@ -74,7 +74,6 @@ resource "aci_function_node" "example" {
 - `conn_consumer_dn` - (Read-Only) Distinguished name of the Function Node consumer connector. Type: String.
 - `conn_provider_dn` - (Read-Only) Distinguished name of the Function Node provider connector. Type: String.
 
-
```

Comment:
Why is this change occurring? This seem to come from legacy docs?

Reply:
The existing file has an extra new line, that is removed by the `go generate` command.